iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergOutputCommitter.java

```diff
@@ -661,6 +669,7 @@ private static FilesForCommit collectResults(int numTasks, ExecutorService execu
     Collection<DataFile> dataFiles = new ConcurrentLinkedQueue<>();
     Collection<DeleteFile> deleteFiles = new ConcurrentLinkedQueue<>();
     Collection<DataFile> referencedDataFiles = new ConcurrentLinkedQueue<>();
+    Collection<CharSequenceSet> referencedDataFilesInDeleteFiles = new ConcurrentLinkedQueue<>();
```

Comment:
why don't we use Collection<CharSequence> and later on transform to needed type?